src/tests/unittests/Makefile.mak

```diff
@@ -1,11 +1,12 @@
 TOPDIR = ..\..\..
 
-TARGETS = asn1 compression pkcs15filter check_macro_reference_loop
+TARGETS = asn1 compression pkcs15filter check_macro_reference_loop strip_pkcs1_2_padding
 
 OBJECTS = asn1.obj \
 	compression.obj \
 	pkcs15-emulator-filter.obj \
-	check_macro_reference_loop.obj
+	check_macro_reference_loop.obj \
+	strip_pkcs1_2_padding.obj
```

Comment:
should this line be terminated with the `\`  so the versioninfo is also part of the objects list?